snuba/api.py

```diff
@@ -364,18 +364,14 @@ def ensure_table_exists(dataset, force=False):
             return
 
         assert local_dataset_mode(), "Cannot create table in distributed mode"
-        from snuba.clickhouse import get_table_definition, get_test_engine
         # We cannot build distributed tables this way. So this only works in local
         # mode.
         clickhouse_rw.execute(
-            get_table_definition(
-                name=dataset.SCHEMA.get_local_table_name(),
-                engine=get_test_engine(),
-            )
+            dataset.get_schema().get_local_table_definition()
         )
 
         from snuba import migrate
-        migrate.run(clickhouse_rw, dataset.SCHEMA.get_table_name())
+        migrate.run(clickhouse_rw, dataset.get_schema().get_table_name())
```

Comment:
There's an inconsistency here (noticed while reviewing #323) where:

- the table is being created (if it doesn't exist already) with the local table name
- the table is being migrated with the distributed or local table name, based on the dataset mode — though at this point we've asserted that we're in local mode, so in practice it can only be the local table name
- the CLI is passing the local table explicitly

Should this be using the local table name explicitly, rather than the `get_table_name` method?